torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1993,6 +1993,28 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad, **kwargs)
     return samples
 
 
+def sample_inputs_matmul(op_info, device, dtype, requires_grad):
+    test_cases = [((L,), (L,), '', (True,)),
```

Comment:
The strings and bools can be removed from this iterable.

Style nit: prefer tuples to lists when the iterable doesn't need to be mutable.